### PR TITLE
Now with updates for Navigation and Table controllers

### PR DESCRIPTION
**Pull request by MikaelNordberg**
Updated with changes from the week 18 workshop together with SSD

### CODE REVIEW

File: PXWeb/Controllers/Api2/NavigationApi.cs [hunk 1]
```diff
@@ -54,24 +54,24 @@ public virtual IActionResult GetNavigationById([FromRoute][Required]string id, [
         /// <response code="200">Success</response>
         /// <response code="429">Error respsone for 429</response>
         [HttpGet]
-        [Route("/v2/navigation")]
+        [Route("/v1/navigation")]
```
Comment (likp) [inline PXWeb/Controllers/Api2/NavigationApi.cs:57]: The spec is incorrect on this. It should be a api/v2 I will fix a change on this.

**Review by likp — APPROVED** [review]
It looks just like the spec. But the spec have some faults :-)

File: PXWeb/Controllers/Api2/NavigationApi.cs (hunk 1 above)
Comment (likp) [reply PXWeb/Controllers/Api2/NavigationApi.cs:57]: Added a PR for this https://github.com/statisticssweden/PxApiSpecs/pull/9

Comment (MikaelNordberg) [reply PXWeb/Controllers/Api2/NavigationApi.cs]: I have updated the routes in the controllers now

**Pull request merged**